src/languages/en.js

```diff
@@ -169,7 +169,6 @@ export default {
         attachmentTooSmall: 'Attachment too small',
         sizeNotMet: 'Attachment size must be greater than 240 bytes.',
         wrongFileType: 'Attachment is the wrong type',
-        notAllowedExtension: 'This filetype is not allowed',
```

Comment:
@youssef-lr `notAllowedExtension` is used in

https://github.com/Expensify/App/blob/7b13f1123301f4783f630aa39af582a4d0dc6a1d/src/libs/ReceiptUtils.js#L11

by https://github.com/Expensify/App/pull/23046 PR which was merge few days back.